modules/serverless-sinkbinding-kn.adoc

```diff
@@ -3,37 +3,32 @@
 // serverless/knstive_eventing/serverless-sinkbinding.adoc
```

Comment:
This was already here and not important, but there is a typo here knstive_eventing -> knative_eventing